e2e/suite/oracle/provider.go

```diff
@@ -113,7 +113,11 @@ func (p *oracleProvider) BeforeEach() {
 		Spec: esv1alpha1.SecretStoreSpec{
 			Provider: &esv1alpha1.SecretStoreProvider{
 				Oracle: &esv1alpha1.OracleProvider{
-					Auth: esv1alpha1.OracleAuth{
+					Region: p.region,
+					Vault:  "vaultOCID",
```

Comment:
Is "vaultOCID" the default value for this field? If it is, maybe we should consider this field as optional and set the default.

Reply:
Actually I don't know if this e2e is even running somewhere, currently (before my change) it seems that the CRD has no vault at all which is not a valid CRD.

Reply:
@knelasevero do you know if this e2e is actually being used somewhere?

Reply:
Implemented, but never called. We don't have a dedicated oracle account to use run them against it